assembl/static2/js/app/components/administration/discussion/editSectionForm.jsx

```diff
@@ -0,0 +1,139 @@
+// @flow
+import React from 'react';
+import { connect } from 'react-redux';
+import { Translate, I18n } from 'react-redux-i18n';
+import { FormGroup, Checkbox, OverlayTrigger, Button } from 'react-bootstrap';
+import { getEntryValueForLocale } from '../../../utils/i18n';
+import FormControlWithLabel from '../../common/formControlWithLabel';
+import { deleteSectionTooltip, upTooltip, downTooltip } from '../../common/tooltips';
+import {
+  updateSectionTitle,
+  updateSectionUrl,
+  toggleExternalPage,
+  deleteSection,
+  upSection,
+  downSection
+} from '../../../actions/adminActions/adminSections';
+
+type EditSectionFormProps = {
+  i18n: Object,
+  locale: string,
+  url: string,
+  type: string,
+  title: string,
+  nbSections: number,
+  index: number,
+  handleTitleChange: Function,
+  handleUrlChange: Function,
+  handleCheckboxChange: Function,
+  handleDeleteClick: Function,
+  handleDownClick: Function,
+  handleUpClick: Function
+};
+
+const EditSectionForm = ({
+  i18n,
+  locale,
+  url,
+  type,
+  title,
+  nbSections,
+  index,
+  handleTitleChange,
+  handleCheckboxChange,
+  handleUrlChange,
+  handleDeleteClick,
+  handleDownClick,
+  handleUpClick
+}: EditSectionFormProps) => {
+  const { translations } = i18n;
+  const titlePh = I18n.t('administration.sections.titlePh');
+  const urlPh = I18n.t('administration.sections.urlPh');
```

Comment:
What Ph means?

Reply:
placeholder (took some time for me to figure it out the first time too)

Reply:
Yeah, for me too the first time, then I forgot! We should really rename this.